### DISCUSSION TITLE
How to use the "Store"  modeul of Connecter in the transfer module of Samples to store something in database?

### DISCUSSION THREAD

**Discussion by 1156458588** [eclipse-edc/Samples#289] [Q&A]
I am a beginner in Java, here，I would like to ask how to implement the following requirements：
As is shown in the picture of the Connector:
![image](https://github.com/user-attachments/assets/4d772ed5-14e4-4471-bebe-cab9991f9e1a)
the Store module could store something like asset in database, so I would like to use it in the transfer module of Samples :
![image](https://github.com/user-attachments/assets/4b75ba7c-b0e1-4caf-a995-d1a176549653)
so that I could store something like asset in my database, I would like to ask you how could I realize it ?

**Comment by paullatzelsperger** [comment]
you will have to prepare the database, i.e. run scripts to create the table structures (`*.sql` files are in the code base), then add all the relevant SQL modules (transaction context, datasource, etc.) and configure them properly. You will find this in the documentation.

We are currently evaluating the implementation of ready-to-use migration modules, which would simplify this process further.

**Reply by 1156458588** [reply]
Thanks for your help very much. Here,I have prepared my database like Postgres,and the *.sql files in the Store module have been executed to create tables, but how can i use the Store module of Connector in the transfer module of Samples to store assets in my Postgres? Could you please provide detailed steps?

**Reply by matinte** [reply]
Hi there! I'm also interested on using Store mode for Transfer, Policy and Federated Catalog. Could you please indicate where are sql files to create tables? Also, did you manage to store data on these tables?

Thanks in advance!
